.vib/joomla/cypress/cypress/integration/joomla_spec.js

```diff
@@ -0,0 +1,72 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('Username and password do not match').should('not.exist');
+  cy.contains('User Menu').click();
+  cy.contains('Log out').click();
+  cy.get("[name='username']");
+});
+
+it('allows creating an article', () => {
+  cy.login();
+  cy.visit('/administrator/index.php?option=com_content&view=articles');
+  cy.get('.button-new').click();
+  cy.fixture('articles').then((article) => {
+    cy.get('#jform_title')
+      .scrollIntoView()
+      .type(`${article.newArticle.title} ${random}`, {
+        force: true,
+      });
+  });
+  cy.contains('Save & Close').click();
+  cy.contains('Article saved');
+});
+
+it('allows updating an image', () => {
+  cy.login();
+  cy.visit('/administrator/index.php?option=com_media');
+  cy.contains('Upload').click();
+  cy.get('input[type="file"]').selectFile(
+    'cypress/fixtures/images/post_image.png',
+    { force: true }
+  );
+  cy.contains('Item uploaded');
+});
+
```

Comment:
One idea to consider, is maybe we can do this as a part of the article creation test, and instead use this test to test the adding user's functionality.
I think it will be valuable because a CMS is usually used by a small business which will have a couple of users, so this would be the first thing they would need to do.